src/controllers/user.js

```diff
@@ -34,7 +34,26 @@ const getSingleUser = async (req, res) => {
   }
 };
 
+const deleteProfile = async (req, res) => {
+  try {
+    const foundUser = await UserModel.findById(req.params.id);
```

Comment:
- The endpoint name could be improved as `/profile/terminate` or `/profile/delete`.
- It shouldn't include `req.params.id`.
```
await UserModel.findByIdAndDelete(req.user.userId);
```
### Front end team can figure a way out and do the confirmation ;)

Reply:
Plus, changing the docs accordingly.

Get well soon ;)